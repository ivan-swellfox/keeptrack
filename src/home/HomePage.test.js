// import React from 'react';
// import { render, screen } from '@testing-library/react';
// import App from '../App';
// import HomePage from './HomePage';
// import { shallow } from 'enzyme';

// test('renders home heading', () => {
//   render(<App />);
//   expect(screen.getByRole('heading')).toHaveTextContent('Home');
// });

// test('renders title tag', () => {
//   let wrapper;
//   wrapper = shallow(<HomePage />);
//   expect(wrapper.exists('h2')).toBeTruthy();
// });
import React from 'react';
import { render, screen } from '@testing-library/react';
import { shallow, ShallowWrapper } from 'enzyme';
import App from '../App';
import HomePage from './HomePage';
import renderer from 'react-test-renderer';

describe('<HomePage />', () => {
  test('should render without crashing', () => {
    render(<App />);
  });

  test('snapshot', () => {
    const tree = renderer.create(<HomePage />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});

describe('<HomePage/>', () => {
  test('renders learn react link', () => {
    render(<HomePage />);
    expect(screen.getByRole('heading')).toHaveTextContent('Home');
  });
  test('snapshot', () => {
    const tree = renderer.create(<HomePage />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
