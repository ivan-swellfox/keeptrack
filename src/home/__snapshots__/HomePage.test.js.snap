// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomePage /> snapshot 1`] = `
<h2>
  Home
</h2>
`;

exports[`<HomePage/> snapshot 1`] = `
<h2>
  Home
</h2>
`;
